tools/build/build.sh

```diff
@@ -48,9 +48,11 @@ fi
 
 if (( $# > 0 )); then
   readonly GIT_ROOT=$(git rev-parse --show-toplevel)
+  # To prevent every build from downloading a fresh copy of gradle, we can keep the downloaded gradle between builds and re-use it
```

Comment:
We shouldn't use /tmp because it won't be cleaned by a clean command. We should keep all input in the same place.
@alalamav where does the build output go? We should probably put the cache there.

Reply:
Cordova uses `platforms/`; Electron, `build/`. We can use the toplevel `build` directory, which would get removed by the clean command.